user/templates/templates.md

```diff
@@ -218,40 +218,58 @@ Please see [How to Reinstall a Template](/doc/reinstall-template/).
 
 ## Switching
 
-When you install a new template or upgrade a clone of a template, it is
-recommended that you switch everything that was set to the old template to the
+When you install a new template or
+[upgrade](/doc/how-to-update/#upgrading-to-avoid-eol) a template, it is
+recommended that you switch everything that was using the old template to the
 new template:
 
-1. Make the new template the default template.
-
-    ```
-    Applications Menu -> System Tools -> Qubes Global Settings -> Default template
-    ```
-
-2. If your keyboard or mouse is connected through `sys-usb`, switch `sys-usb`
-   to the new template. (Note that this is a single command to ensure that
-   `sys-usb` restarts. If it does not, you will not be able to use your USB
-   keyboard or mouse.)
-
-    ```
-    [user@dom0 ~]$ qvm-shutdown --wait sys-usb; qvm-prefs sys-usb template <NEW_TEMPLATE>; qvm-start sys-usb
-    ```
-
-3. Base app qubes on the new template.
-
-    ```
-    Applications Menu -> System Tools -> Qubes Template Manager
-    ```
-
-4. Base the [disposable template](/doc/glossary/#disposable-template) on the new
-   template.
-
-    ```
-    [user@dom0 ~]$ qvm-create -l red -t <NEW_TEMPLATE> <NEW_DISPOSABLE_TEMPLATE>
-    [user@dom0 ~]$ qvm-prefs <NEW_DISPOSABLE_TEMPLATE> template_for_dispvms True
-    [user@dom0 ~]$ qvm-features <NEW_DISPOSABLE_TEMPLATE> appmenus-dispvm 1
-    [user@dom0 ~]$ qubes-prefs default-dispvm <NEW_DISPOSABLE_TEMPLATE>
-    ```
+1. Make the new or upgraded template the default template. In the App Menu, go
+   to Qubes Tools, then click on Qubes Global Settings. In the Qube Defaults
+   section, next to Template, select the new or upgraded template from the
+   drop-down list. Press OK.
+
+2. Base your [disposable templates](/doc/glossary/#disposable-template) on the
+   new template. In the App Menu, go to Qubes Tools, then click on Qube
+   Manager. In the Qube Manager, find your disposable templates. (By default,
+   they end in `-dvm`.) Right click, then click on Settings. Next to Template,
```

Comment:
You can change template directly from right-click menu - there is "template" submenu.

Reply:
Discovered a problem with this method: https://github.com/QubesOS/qubes-issues/issues/8074.

Reply:
Ok, I think I was able to rewrite the instructions in a way that avoids triggering this bug.